Hunting Queries/Microsoft 365 Defender/Cloud Apps/file-download-events.yaml

```diff
@@ -0,0 +1,20 @@
+id: f8f8d4a5-7d7d-4c5d-9b5c-9c5d7d8f8f8f
+
+name: File download events in the last 7 days
+
+description: |
+  This query looks for file download events identified by Microsoft Defender for Cloud Apps. It will require an corresponding app connector in Microsoft Defender for Cloud Apps.
+  Reference - https://learn.microsoft.com/en-us/defender-cloud-apps/enable-instant-visibility-protection-and-governance-actions-for-your-apps
+  Reference - https://github.com/Azure/Azure-Sentinel/blob/master/Hunting%20Queries/Cloud Apps/file-download-events.yaml
```

Comment:
Hi @Keithbfleming, second reference link isn't working.

Reply:
@v-rbajaj it's a link to the yaml itself, I wouldn't expect it to work until committed in the master branch

Reply:
I think you are trying to reference this link - https://github.com/Azure/Azure-Sentinel/blob/master/Hunting%20Queries/Microsoft%20365%20Defender/Cloud%20Apps/file-download-events.yaml